app/services/account_getter.go

```diff
@@ -1,11 +1,47 @@
 package services
 
 import (
+	"time"
+
 	"github.com/keratin/authn-server/app/data"
 	"github.com/keratin/authn-server/app/models"
 	"github.com/pkg/errors"
 )
 
+func AccountToJson(account *models.Account) map[string]interface{} {
```

Comment:
Don't think this makes a ton of sense in services.  Maybe implementing `json.Marshaler` on the account and oauth account models would make sense?

https://pkg.go.dev/encoding/json#Marshaler

Reply:
Nice one! Account and oauth account now implements Marshaler interface.